.circleci/config.yml

```diff
@@ -33,20 +33,22 @@ jobs:
       DBT_INVOCATION_ENV: circle
     docker:
       - image: fishtownanalytics/test-container:10
-      - image: godatadriven/spark:2
+      - image: godatadriven/spark:3.0
```

Comment:
You might consider trying with 3.1 or 3.2. There were _absolutely_ some bugs in Spark 3.0.0, but I'm admittedly somewhat distrustful of the whole of Spark 3.0.x.

Just throwing that out there in case you're still stuck.